modules/migration-mtc-release-notes-1-7-09.adoc

```diff
@@ -0,0 +1,22 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-09_{context}"]
+= {mtc-full} 1.7.9 release notes
+
+[id="resolved-issues-1-7-09_{context}"]
+== Resolved issues
+
+There are no major resolved issues in this release.
+
+
+[id="known-issues-1-7-09_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Adjust rsync options in DVM
+
+In this release, users are unable to prevent absolute symlinks from being manipulated by Rsync during direct volume migration (DVM). (link:https://bugzilla.redhat.com/show_bug.cgi?id=2204461[*BZ#2204461*])
```

Comment:
```suggestion
In this release, users are unable to prevent absolute symlinks from being manipulated by rsync during direct volume migration (DVM). (link:https://bugzilla.redhat.com/show_bug.cgi?id=2204461[*BZ#2204461*])
```